webapp/channels/src/components/admin_console/ldap_wizard/ldap_wizard.tsx

```diff
@@ -173,6 +206,7 @@ const LDAPWizard = (props: Props) => {
     const buildButtonSetting = (setting: AdminDefinitionSetting | LDAPDefinitionSettingButton) => {
         let config = JSON.parse(JSON.stringify(props.config));
         config = getConfigFromState(config, state, schema, isDisabled);
+        const filterTestResultsHandler = setting.key === 'LdapSettings.TestFilters' ? handleFilterTestResults : undefined;
```

Comment:
Not sure if this is worth changing now, but this does look more like we should possibly be using a `custom` setting for the test button. I'm not entirely sure how the existing buttons work, but I guess they just have a single error instead of spreading them out through the inputs which is a much nicer way to do things